### PR TITLE
Security hub region parameter, DB table name chnage.

### PR DESCRIPTION
**Pull request by danielmkn**
DB table name vary depending on the environment.
Use AWS Security Hub deployment parameter to use it in import and update calls.

### CODE REVIEW

File: transformer/app.js [hunk 1]
```diff
@@ -4,9 +4,11 @@ import { DynamoDBClient, QueryCommand } from '@aws-sdk/client-dynamodb';
 import { getLogger } from './logger.js';
 
 const logger = getLogger();
+const region = process.env.AWS_REGION;
```
Comment (alexhung) [inline transformer/app.js:7]: This variable is redundant since it is only used once in the next line. Just inline it.

File: transformer/app.js [hunk 2]
```diff
@@ -4,9 +4,11 @@ import { DynamoDBClient, QueryCommand } from '@aws-sdk/client-dynamodb';
...
+const securityHubRegion = process.env.SECURITY_HUB_REGION || region;
```
Comment (alexhung) [inline transformer/app.js:8]: This should be UPPERCASE

File: transformer/app.js [hunk 3]
```diff
@@ -128,10 +128,12 @@ const generateUpdatePayload = (existingFindingsToUpdate) => existingFindingsToUp
   },
 }));
 
+const xrayFindingsTable = process.env.XRAY_FINDINGS_TABLE;
```
Comment (alexhung) [inline transformer/app.js:131]: This should have a fallback value. And the variable should be UPPERCASE

**Review by alexhung — CHANGES_REQUESTED** [review]

File: transformer/app.js (hunk 1 above)
Comment (danielmkn) [reply transformer/app.js:7]: It's also used on lines 75 and 79. We might remove 79 on the next PR. But anyway, the var should stay.

File: transformer/app.js (hunk 2 above)
Comment (danielmkn) [reply transformer/app.js:8]: fixed

File: transformer/app.js (hunk 1 above)
Comment (danielmkn) [reply transformer/app.js:7]: and uppercased.

File: transformer/app.js [hunk 4]
```diff
@@ -4,9 +4,11 @@ import { DynamoDBClient, QueryCommand } from '@aws-sdk/client-dynamodb';
 import { getLogger } from './logger.js';
 
 const logger = getLogger();
+const REGION = process.env.AWS_REGION;
+const SECURITY_HUB_REGION = process.env.SECURITY_HUB_REGION || REGION;
 
 const ddbClient = new DynamoDBClient();
-const hubClient = new SecurityHubClient();
+const hubClient = new SecurityHubClient({ region: SECURITY_HUB_REGION });
```
Comment (alexhung) [inline transformer/app.js:11]: I wonder if this is necessary. Probably don't do any harm.

**Review by alexhung — APPROVED** [review]

File: transformer/app.js (hunk 4 above)
Comment (danielmkn) [reply transformer/app.js:11]: It is necessary. https://jira.jfrog.org/browse/PTRENG-4401

Comment (alexhung) [reply transformer/app.js:11]: That ticket, the way I read it from Scott's comment during the meeting, is that the region needs to be part of the integration ARN. I don't know if this region needs to be set for the SDK client.

**Pull request merged**